O365/utils/token.py

```diff
@@ -44,6 +44,26 @@ def expiration_datetime(self):
             expires_on = expires_on + dt.timedelta(days=90)
         return expires_on
 
+    @property
+    def access_expiry(self):
```

Comment:
To maintain the same naming convention this should be called `access_expiration_datetime`.

Reply:
sure...done